### PR TITLE
Skipping the CI secret sauce 🦕

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

This is related to https://github.com/tektoncd/pipeline/issues/4220. Not fixing it but skipping the problematic tests for now to unblock the CI for the rest of contributors while we fix it.

/hold

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [x] [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [x] Release notes block below has been filled in or deleted (only if no user facing changes)

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by vdemeester** [comment]
/area test
This is related to https://github.com/tektoncd/pipeline/issues/4220. Not fixing it but skipping the problematic tests for now to unblock the CI for the rest of contributors while we fix it.

File: examples/v1beta1/taskruns/foo.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
```
Comment (chmouel) [inline examples/v1beta1/taskruns/foo.yaml:1]: did you want to commit this?

**Comment by chmouel** [comment]
i think you commited extra files

File: examples/v1beta1/taskruns/foo.yaml (hunk 1 above)
Comment (vdemeester) [reply examples/v1beta1/taskruns/foo.yaml:1]: oups..

**Comment by chmouel** [comment]
/lgtm

Looking Good! 🚀

**Comment by afrittoli** [comment]
The `kind/failing-test` label is not known by the CI job, since it's not configured as a valid label, which is why the CI job fails:
- valid labels: https://github.com/tektoncd/plumbing/blob/main/label_sync/labels.yaml
- CI Job error: https://dashboard.dogfooding.tekton.dev/#/namespaces/tektonci/pipelineruns/check-pr-labels-2qks5?pipelineTask=kind-label&step=check-labels

File: internal/builder/v1beta1/pod.go [hunk 2]
```diff
@@ -12,6 +12,7 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
+
```
Comment (afrittoli) [inline internal/builder/v1beta1/pod.go:15]: Was this added by accident?

Comment (vdemeester) [reply internal/builder/v1beta1/pod.go:15]: ah that was to make sure it failed before.. so.. yes and no 😛 I can revert 😬

**Comment by chmouel** [comment]
/lgtm

Looking Good! 🤙🏽

**Review by afrittoli — COMMENTED** [review]
This could close #4220 and we could have another issue about restoring the disabled tests, but it's also ok to continue on #4220

/approve

**Pull request merged**